bullet_train/app/helpers/account/dates_helper.rb

```diff
@@ -1,17 +1,54 @@
 module Account::DatesHelper
-  def display_date(timestamp, format: :default, date_format: nil)
+  def display_date(timestamp, custom_date_format = nil, format: :default, date_format: nil)
```

Comment:
I added back `custom_date_format` to preserve the call signature to that we don't cause errors if someone is calling this method directly with the old syntax.

Reply:
Good call!